### PR TITLE
Remove empty inheritDoc's + format all files

### PR DESCRIPTION
**Pull request by joaodinissf**

### CODE REVIEW

File: com.avaloq.tools.ddk.checkcfg.ui/src/com/avaloq/tools/ddk/checkcfg/ui/outline/CheckCfgOutlineTreeProvider.java [hunk 1]
```diff
@@ -52,7 +52,7 @@ protected Object _text(final ConfiguredCatalog configuredCatalog) {
     if (configuredCatalog.getCatalog() == null || configuredCatalog.getCatalog().eIsProxy()) {
       // Unresolved reference
       ruleText = new StyledString("<Unresolved catalog>", //
-      stylerFactory.createXtextStyleAdapterStyler(semanticHighlightingConfiguration.errorTextStyle()));
+          stylerFactory.createXtextStyleAdapterStyler(semanticHighlightingConfiguration.errorTextStyle()));
```
Comment (rubenporras) [inline com.avaloq.tools.ddk.checkcfg.ui/src/com/avaloq/tools/ddk/checkcfg/ui/outline/CheckCfgOutlineTreeProvider.java:55]: Indentation on this file seems wrong now

File: com.avaloq.tools.ddk.sample.helloworld/dsl-gen/com/avaloq/tools/ddk/sample/helloworld/validation/ExecutionEnvironmentStandaloneSetup.java [hunk 2]
```diff
@@ -18,7 +18,7 @@ public class ExecutionEnvironmentStandaloneSetup implements ICheckValidatorStand
   private static final String GRAMMAR_NAME = "com.avaloq.tools.ddk.sample.helloworld.HelloWorld";
   private static final String CATALOG_FILE_PATH = "com/avaloq/tools/ddk/sample/helloworld/validation/ExecutionEnvironment.check";
 
-  @Override
+  /** {@inheritDoc} */
```
Comment (rubenporras) [inline com.avaloq.tools.ddk.sample.helloworld/dsl-gen/com/avaloq/tools/ddk/sample/helloworld/validation/ExecutionEnvironmentStandaloneSetup.java:21]: the @override should stay, shouldn't it?

**Review by rubenporras — CHANGES_REQUESTED** [review]

File: com.avaloq.tools.ddk.sample.helloworld/dsl-gen/com/avaloq/tools/ddk/sample/helloworld/validation/ExecutionEnvironmentStandaloneSetup.java (hunk 2 above)
Comment (joaodinissf) [reply com.avaloq.tools.ddk.sample.helloworld/dsl-gen/com/avaloq/tools/ddk/sample/helloworld/validation/ExecutionEnvironmentStandaloneSetup.java]: Correct, two `dsl-gen` files were modified by mistake in this plugin. Fixed now.

File: com.avaloq.tools.ddk.checkcfg.ui/src/com/avaloq/tools/ddk/checkcfg/ui/outline/CheckCfgOutlineTreeProvider.java (hunk 1 above)
Comment (joaodinissf) [reply com.avaloq.tools.ddk.checkcfg.ui/src/com/avaloq/tools/ddk/checkcfg/ui/outline/CheckCfgOutlineTreeProvider.java:55]: This indentation is due to the empty comment in the line above.

**Review by rubenporras — APPROVED** [review]

**Pull request merged**